src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+
+    }

```

Comment:
Creating a figure, don't pass expressions in the constructor. To improve readability, assign the 'side' to a variable before passing it to the constructor.